src/obsidian_ai_hub/coding/orchestrator.py

```
@@ -100,20 +106,79 @@ async def generate_response(
+            target_ids = [t["tool_id"] for t in resolved_tool_ids]
+        else:
+            target_ids = self.tool_ids
+
+        trusted_ctx = {
+            "repo_path": repo_path,
+            "backend_name": backend_name,
+        }
+
+        try:
+            allowed_tools = registry.resolve_tools_with_context(target_ids, trusted_ctx)
+        except Exception:
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `Exception`

(BLE001)